tests/storage/hotplug.go

```diff
@@ -1156,17 +1179,23 @@ var _ = SIGDescribe("Hotplug", func() {
 	Context("virtctl", func() {
 		var (
 			vm *v1.VirtualMachine
+			sc string
 		)
 
 		BeforeEach(func() {
+			var exists bool
+			sc, exists = tests.GetStorageClassWithBindingModeWaitForFirstConsumer()
```

Comment:
I think you might have missed the `I don't think we need this function` on GetStorageClassWithBindingModeWaitForFirstConsumer we should pick the RWOFilesystem storage class, and check if it is WFFC, if so run the test, if not skip it.